CMakeLists.txt

```diff
@@ -22,7 +28,11 @@ include(FeatureSummary)
 include(MetalShaderSupport)
 include(VcpkgToolchain)
 
-project(Plasma)
+if(APPLE)
+    project(Plasma LANGUAGES C CXX Swift)
```

Comment:
Would be nice to do a check for Swift support before enabling this (selfishly saying so because this will blow up spectacularly on OS X Leopard)

Reply:
You can also do this by calling `enable_language(Swift)` at a later point in the file too.

```cmake
include(CheckLanguage)

if(APPLE)
    check_language(Swift)
    if(CMAKE_Swift_COMPILER)
        enable_language(Swift)
    else()
        message(WARNING "Swift is not supported")
    endif()
endif()